docker-compose.e2e-compatibility.yml

```diff
@@ -0,0 +1,278 @@
+version: '3.0'
+services:
+  morqa:
+    image: mysteriumnetwork/quality-oracle:latest
+    command: >
+      --bind-addr=:8085
+      --prometheus=http://prometheus:9090
+      --metrics-addr=:8080
+    depends_on:
+      - prometheus
+    expose:
+      - 8085
+      - 8080
+
+  prometheus:
+    image: prom/prometheus:v2.10.0
+    volumes:
+      - ./e2e/prometheus:/prometheus
+    command:
+      - '--config.file=/prometheus/prometheus-local.yml'
+      - '--storage.tsdb.path=/data'
+    expose:
+      - 9090
+
+  ipify:
+    image: owlab/ipify
+    expose:
+      - 3000
+
+  broker:
+    image: nats:1.0.4
+    expose:
+      - 4222
+      - 8222
+
+  db:
+    image: percona:5.7
+    restart: always
+    expose:
+      - 3306
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: myst_api
+      MYSQL_USER: myst_api
+      MYSQL_PASSWORD: myst_api
+
+  mysterium-api:
+    image: mysteriumnetwork/mysterium-api:latest
+    expose:
+      - 8001
+    environment:
+      APP_PORT: 8001
+      DB_HOST: db
+      DB_NAME: myst_api
+      DB_USER: myst_api
+      DB_PASSWORD: myst_api
+      ETHER_RPC_URL: "ws://ganache:8545"
+      IDENTITY_CONTRACT: "0x1955141ba8e77a5B56efBa8522034352c94f77Ea"
+      ETHER_MINING_MODE: "poa"
+      DISCOVERY_VERIFY_IDENTITY: "false"
+    depends_on:
+      - db
+      - ganache
+
+  #private blockchain
+  ganache:
+    image: trufflesuite/ganache-cli:v6.8.1-beta.0
+    expose:
+      - 8545
+    command: >
+      --mnemonic "amused glory pen avocado toilet dragon entry kitchen cliff retreat canyon danger"
+
+  transactor:
+    image: mysteriumnetwork/transactor:latest
+    environment:
+      PORT: 8888
+    expose:
+      - 8888
+    depends_on:
+      - ganache
+    command: >
+      -mystSCAddress 0x4D1d104AbD4F4351a0c51bE1e9CA0750BbCa1665
+      -rpcEndpoint ws://ganache:8545
+      -registry 0xbe180c8CA53F280C7BE8669596fF7939d933AA10
+      -ourIdentity 0x354bd098b4ef8c9e70b7f21be2d455df559705d7
+      -chImplementation 0x599d43715DF3070f83355D9D90AE62c159E62A75
+      -balanceCheckInterval 1s
+    volumes:
+      - ./e2e/blockchain/keystore:/keystore
+
+  accountant:
+    image: mysteriumnetwork/accountant:latest
+    environment:
+      PORT: 8889
+    expose:
+      - 8889
+    depends_on:
+      - ganache
+    command: >
+      -identityPassphrase ""
+      -myst 0x4D1d104AbD4F4351a0c51bE1e9CA0750BbCa1665
+      -rpcEndpoint ws://ganache:8545
+      -registry 0xbe180c8CA53F280C7BE8669596fF7939d933AA10
+      -operator 0x354bd098b4ef8c9e70b7f21be2d455df559705d7
+      -chImplementation 0x599d43715DF3070f83355D9D90AE62c159E62A75
+      -keystoreDir /keystore
+      -accountantAddress 0x241F6e1d0bB17f45767DC60A6Bd3D21Cdb543a0c
+      -boltPath /bolt.db
+    volumes:
+      - ./e2e/blockchain/keystore:/keystore
+
+  myst-provider:
+    image: mysteriumnetwork/myst:0.19.1-alpine
```

Comment:
Are we testing compatibility with `0.19.1` only? 
What is the flow to change the version that should be tested for compatibility?

Reply:
Actually there is no need to test with different version if all new versions are compatible with this fixed one we are fine. Aren't we?

Reply:
Ok, I see the plan. Let's see how it will work.